css-pseudo-4/Overview.bs

```diff
@@ -667,12 +689,20 @@ Cascading and Per-Element Highlight Styles</h3>
   for which that element is the <a>originating element</a>.
   When multiple styles conflict,
   the winning style is determined by the cascade.
+
   When any supported property is not given a value by the cascade,
-  it's value is determined by inheritance from
+  or given a value of ''inherit'',
```

Comment:
Say “''inherit'' or ''unset''” here, and then delete the new paragraph below.

Reply:
done in 06de02d057ea554fac09643037878d426b9941fb